frontend/src/lib/api/candidate.ts

```diff
@@ -181,6 +225,18 @@ export const getExistingAnswers = async (): Promise<Response | undefined> => {
   return res;
 };
 
+export const getLanguages = async (): Promise<StrapiLanguageData[] | undefined> => {
+  const res = await request(
+    getUrl('api/languages', {
+      'populate[language]': 'true'
+    })
+  );
+  if (!res?.ok) return [];
```

Comment:
We probably would want to return undefined over an empty array if fetching languages fail, so it's explicitly clear that fetching languages failed compared to just there not being any languages.